enterprise/backend/test/metabase_enterprise/internal_user_test.clj

```diff
@@ -1,10 +1,13 @@
 (ns metabase-enterprise.internal-user-test
   (:require
-   [clojure.test :refer [deftest is]]
+   [clojure.string :as str]
+   [clojure.test :refer [deftest is testing]]
```

Comment:
nit: not sure why this was done this way originally but we usually just `:refer :all` for `clojure.test`